scwx-qt/source/scwx/qt/map/overlay_layer.cpp

```diff
@@ -143,7 +144,10 @@ class OverlayLayerImpl
    float    lastFontSize_ {0.0f};
    QMargins lastColorTableMargins_ {};
 
-   double cursorScale_ {1};
+   double                             cursorScale_ {1};
+   boost::signals2::scoped_connection cursorScaleConnection_;
```

Comment:
I think my assertion that making this the last member in the class will make this thread safe, is not correct.

cursorScaleConnection_.disconnect() should be called in the destructor.